rest/src/main/java/fi/otavanopisto/pyramus/rest/CourseRESTService.java

```diff
@@ -284,6 +298,11 @@ public Response updateCourse(@PathParam("ID") Long id, fi.otavanopisto.pyramus.r
       }
     }
     
+    Organization organization = organizationController.findById(courseEntity.getOrganizationId());
```

Comment:
Check that the logged user has access to the organization (Userutils.smth). Should we also check that the user has access to the previous (if different) organization?

Reply:
I would add the check that the user has access to the previous organization (course.getOrganization()) too as well as the new organization below just to be sure.